jobs/redis-blacksmith-plans/templates/plans/cluster/manifest.yml

```diff
@@ -26,7 +26,7 @@ instance_groups:
   - name: node
     instances: (( grab meta.instances ))
     azs: (( grab meta.azs || meta.default.azs ))
-    networks: [name: (( grab meta.net || 'redis-service' ))]
+    networks: [name: (( grab meta.net || "redis-service" ))]
```

Comment:
Are these quoting changes strictly necessary?

Reply:
When I ran the following on play.spruce.cf  I got the following error.
---
meta:
  env1: prod

junk: (( grab meta.env || 'default' ))

1 error(s) detected:
  - $.junk: Unable to resolve `'default'`: `$.'default'` could not be found in
   → the datastructure

When I change to double quotes, it works.

So this change seems to be needed.